rector.php

```diff
@@ -8,7 +8,7 @@
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->paths([
         __DIR__ . '/app',
-        __DIR__ . '/domain',
+        __DIR__ . '/domain/src',
```

Comment:
ℹ️ Missing from a [previous PR](https://github.com/osteel/dime/pull/23) – we don't need to refactor the tests